launch/base_hardware.launch

```diff
@@ -1,13 +1,13 @@
 <launch>
-    <arg name="sim" default="false" doc="true if we are running in the sim" />
+    <arg name="platform" default="sim" doc="string representing the platform" />
     <arg name="bond_id_namespace" default="safety_bonds" />
 
     <!-- if we're in the sim -->
-    <group if="$(arg sim)">
+    <group if="$(eval arg('platform') == 'sim')" />
         <include file="$(find iarc7_simulator)/launch/morse.launch" />
     </group>
-    <!-- else -->
-    <group unless="$(arg sim)">
+    <!-- if in V1.1 -->
+    <group if="$(eval arg('platform') == '1.1')" />
```

Comment:
Is there a reason this isn't just `if not in sim` like it was before?

Reply:
Yes, I'm about to add another target that will launch different hardware.

Reply:
The other target is the crazyflie, right?  Should this be `if not sim and not crazyflie`?

Reply:
i.e. `if not sim` for now, and add the `if not crazyflie` when that's ready?

Reply:
That doesn't make sense because the hardware being launched is specific to V1.1

Also the crazyflie PR is coming tonight so I wouldn't make changes based on that.

Reply:
Ok that's fine then.  I was thinking this would be mostly common between revs of the main drone, but enough is different we can do it separately.